packages/core/src/marine/deps-loader/common.ts

```diff
@@ -0,0 +1,24 @@
+import { BlobWorker } from 'threads';
+import { fromBase64, toUint8Array } from 'js-base64';
+import type { WorkerImplementation } from 'threads/dist/types/master';
+import { LazyLoader } from '../../interfaces/index.js';
+import { Buffer } from 'buffer';
+
+export class InlinedWorkerLoader extends LazyLoader<WorkerImplementation> {
+    constructor(b64script: string) {
+        super(() => {
+            const script = fromBase64(b64script);
+            return Promise.resolve(BlobWorker.fromText(script));
```

Comment:
I think it should be fine to just leave it like this
```suggestion
            return BlobWorker.fromText(script);
```